test/controllers/architectures_controller_test.rb

```diff
@@ -80,4 +80,17 @@ def user_with_viewer_rights_should_succeed_in_viewing_architectures
     get :index
     assert_response :success
   end
+
+  # randomly selected architectures controller for the following test
+  def should_render_pagination_when_needed
+    old = Setting[:entries_per_page]
+    Setting[:entries_per_page] = Architecture.count
```

Comment:
can you ensure there is at least one ? i see a difference between:
- no entries at all (we should see the patternfly empty state view)
- entries < default pagination size
- entries > default pagination size